src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  fahrenheit: 60,
+  city: 'Edmonds',
+  sky: 'Sunny',
+};
+let num = state.fahrenheit;
+const tempUp = (event) => {
+  state.fahrenheit += 1;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const increase = document.getElementById('f');
+  increase.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return increase.textContent;
+};
+
+const tempDown = (event) => {
+  state.fahrenheit--;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const decrease = document.getElementById('f');
+  decrease.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return decrease.textContent;
```

Comment:
Notice lines 18-22 are almost identical to lines 9-13 above. You can put these 5 lines into a helper method like updateTempAndDisplay() and then call that helper method in the tempUp and tempDown methods.